incubation.md

```diff
@@ -0,0 +1,129 @@
+# Jupyter Incubation Process
+
+Incubation is the process by which new Jupyter Subprojects can be created and evaluated for
```

Comment:
pushed the->a

On Fri, Sep 4, 2015 at 1:25 PM, Kyle Kelley <notifications@github.com>
wrote:

> In incubation.md
> <https://github.com/jupyter/governance/pull/3#discussion_r38790621>:
>
> > @@ -0,0 +1,129 @@
> > +# Jupyter Incubation Process
> > +
> > +Incubation is the process by which new Jupyter Subprojects can be created and evaluated for
>
> Ditto on @minrk <https://github.com/minrk>'s comments about the precedent
> the has vs a here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/governance/pull/3/files#r38790621>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com